File: Learn C#/20230725/MyBankApp/Models/TDBContext.cs
using System.Collections.Generic;
using System.Data.Entity;

namespace MyBankApp.Models
{
    public class TDBContext : DbContext
    {
        public DbSet<Transaction> Transactions { get; set; }
        public DbSet<BankAccount> BankAccounts { get; set; }

        // DbContextを構成するためのコンストラクタ
/*        public TDBContext(DbContextOptions<TDBContext> options) : base(options)
        {
        }*/

    }
}

File: Learn C#/20230728/MyBankAppvb/Models/TDBContext.vb
Imports System.Collections.Generic
Imports System.Data.Entity
Imports MyBankAppvb.MyBankAppvb.Models

Namespace MyBankAppvb.Models
    Public Class TDBContext
        Inherits DbContext

        Public Property Transactions As DbSet(Of Transaction)
        Public Property BankAccounts As DbSet(Of BankAccount)

    End Class
End Namespace
